src/System.Windows.Forms.Design/src/System/ComponentModel/Design/CollectionEditor.cs

```diff
@@ -93,59 +90,52 @@ protected virtual void CancelChanges()
     /// </summary>
     protected virtual object CreateInstance(Type itemType)
     {
-        if (Context.TryGetService(out IDesignerHost host) && typeof(IComponent).IsAssignableFrom(itemType))
+        if (Context.TryGetService(out IDesignerHost? host) && typeof(IComponent).IsAssignableFrom(itemType))
         {
-            IComponent instance = host.CreateComponent(itemType, null);
+            //TODO: change to IComponent instance = host.CreateComponent(itemType) ?
+            IComponent instance = host.CreateComponent(itemType, null!);
```

Comment:
Should we call 
```suggestion
            IComponent instance = host.CreateComponent(itemType);
```
instead?